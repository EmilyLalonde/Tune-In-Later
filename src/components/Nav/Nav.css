
nav {
  display: flex;
  height: 10vh;
  justify-content: space-between; 
  align-items: center;
  padding-top: 20px;
}

a.Nav {
  text-decoration: none;
  color: white;
}

h1 {
  color: #ee8d60;
  font-size: 46px;
  margin: 0;
  margin-left: 1%
}

