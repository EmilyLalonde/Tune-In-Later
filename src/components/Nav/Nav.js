import './Nav.css'
import SearchForm from '../SearchForm/SearchForm';
import React from 'react';
import { NavLink } from 'react-router-dom';

const Nav = () => {
    return (
        <nav>
            <h2>TuneIn!(Later)</h2>
            <NavLink to='/' className='Nav'>Home</NavLink>
            <NavLink to='/favorites' className='Nav'>Favorites</NavLink>
            <NavLink to='/login' className='Nav'>Log In</NavLink>
            <NavLink to='/logout' className='Nav'>Log Out</NavLink>
            <SearchForm />
        </nav>
    )
}

export default Nav