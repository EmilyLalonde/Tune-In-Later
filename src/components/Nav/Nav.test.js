import React from 'react';
import { shallow } from 'enzyme'
import Nav from './Nav';

describe('Nav', () => {

  let wrapper;
  beforeEach(() => {
  const mockCurrentUser = {id: 1, name: 'Joe Tester', email: 'joe@email.com'};
  const mockLogoutUser = jest.fn();

    wrapper = shallow(<Nav 
      currentUser={mockCurrentUser}
      handleLogout={mockLogoutUser}
    />)
  });

  it('should match the snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
})