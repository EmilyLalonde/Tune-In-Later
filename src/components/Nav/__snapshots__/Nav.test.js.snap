// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Nav should match the snapshot 1`] = `
<nav>
  <h1>
    TuneIn!(Later)
  </h1>
  <NavLink
    className="Nav"
    to="/"
  >
    Home
  </NavLink>
  <NavLink
    className="Nav"
    to="/favorites"
  >
    Favorites
  </NavLink>
  <NavLink
    className="Nav"
    to="/login"
  >
    Log In
  </NavLink>
  <NavLink
    className="Nav"
    to="/create-user"
  >
    Create Account
  </NavLink>
  <button
    onClick={[MockFunction]}
  >
    Sign Out
  </button>
  <SearchForm />
</nav>
`;
